website/content/v1.7/talos-guides/install/single-board-computers/bananapi_m64.md

```diff
@@ -36,16 +39,24 @@ In this example, we will assume `/dev/mmcblk0`.
 Now `dd` the image to your SD card:
 
 ```bash
-sudo dd if=metal-bananapi_m64-arm64.img of=/dev/mmcblk0 conv=fsync bs=4M
+sudo dd if=metal-arm64.raw of=/dev/mmcblk0 conv=fsync bs=4M
 ```
 
 ## Bootstrapping the Node
 
 Insert the SD card to your board, turn it on and wait for the console to show you the instructions for bootstrapping the node.
-Following the instructions in the console output to connect to the interactive installer:
+
+Create a `installer-patch.yaml` containing reference to the `installer` image generated from an overlay:
+
+```yaml
+# installer-patch.yaml
+machine:
+  install:
+    image: factory.talos.dev/installer/f48c47c8a27248bc19a878c086f2cb2ed0d5aff777688af95fc88f100d1048e1:{{< release >}}
+```
 
 ```bash
-talosctl apply-config --insecure --mode=interactive --nodes <node IP or DNS name>
+talosctl apply-config --insecure --mode=interactive --config-patch=@installer-patch.yaml --nodes <node IP or DNS name>
```

Comment:
do we support that? :scream: 

interactive + config patch?

Reply:
lol, i hope not, I'll fix it and drop the interactive stuff